svelte.config.js

```diff
@@ -12,15 +11,7 @@ const config = {
 		adapter: adapter(),
 
 		// hydrate the <div id="svelte"> element in src/app.html
-		target: '#svelte',
-
-		vite: {
-			resolve: {
-				alias: {
-					'@components': path.resolve('./src/lib/components')
-				}
-			}
-		}
```

Comment:
Why remove this?

Reply:
I removed this for now as I got some problems with resolving the path. I will give it a try again after I have now figured out everything else to see if it works.

Reply:
Added a dynamic components path.